src/pybind/mgr/dashboard/frontend/src/styles/defaults/_bootstrap-defaults.scss

```diff
@@ -78,14 +83,21 @@ $chart-color-red: $red !default;
 $chart-color-blue: #06c !default;
 $chart-color-orange: #ef9234 !default;
 $chart-color-yellow: #f6d173 !default;
+$chart-color-translucent-red: $red-dim !default;
+$chart-color-blue: $blue !default;
+$chart-color-orange: $orange !default;
+$chart-color-translucent-orange: $orange-dim !default;
```

Comment:
Why are these being redefined?

Reply:
@Pegonzal , where are these being defined before?

Reply:
This was marked as resolved, but I didn't get a change to answer and it hasn't been resvoled either.

So... answering your question @aaSharma14 :
`$chart-color-blue` and `$chart-color-orange` are first defined three lines above where they are being defined again with another color. 

So I don't think we have to set these again, and not sure why are we changing their color